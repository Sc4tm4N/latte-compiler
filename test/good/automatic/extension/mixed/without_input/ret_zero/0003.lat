
int[] node(int node) {
   int[] a = new int[node];
   a[0] = 96;
   return a;
}

class node {
  int node;

  int node() {
     return node;
  }

  int test() {
    return self.node();
  }

  int[] test2() {
    return node(node);
  }
}

int main() {
  node node = new node;
  node.node = 12;
  node.node = node.node();
  printInt(node.node());
  int[] a = node(node.node);
  printInt(a[0]);
  printInt(a[1]);
  return 0;
}
