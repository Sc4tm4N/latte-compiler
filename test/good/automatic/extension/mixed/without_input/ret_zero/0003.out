12
96
0
